### PR TITLE
setup project

### PR DESCRIPTION
**Pull request by munyanezaarmel**
### Brief summary of the change made

setting up DSH project

### Pull Request checklist

Please confirm you have completed any of the necessary steps below.

- [x] Meaningful Pull Request title and description
- [x] Changes tested as described above
- [x] Added appropriate documentation for the change.
- [x] Created GitHub issues for any relevant followup/future enhancements if appropriate.

### CODE REVIEW

**Comment by dmohns** [comment]
Hey @munyanezaarmel 

Thanks a lot 🙏 In this project we don't need the `website` sub-folder. We used this in `OpenPAYGO-js` as the repo was mainly about a JS library.

However, this repo ([digitalization-support-hub](https://github.com/EnAccess/digitalization-support-hub)` is entirely dedicated to the DSH website, so you can move the content of `website` to the root of the repo. (remember to adapt the Github actions accordingly)

**Comment by munyanezaarmel** [comment]
@dmohns hope we are ready to go now

File: .github/workflows/deploy-gh-pages.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+      - name: Setup Pages
+        uses: actions/configure-pages@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "18"
+          cache: npm
+          cache-dependency-path: package-lock.json # Updated to root-level package-lock.json
```
Comment (dmohns) [inline .github/workflows/deploy-gh-pages.yml:38]:
```suggestion
          cache-dependency-path: package-lock.json
```

**Review by dmohns — APPROVED** [review]
Looks good! Thanks a lot 🙏

**Pull request merged**